### ISSUE TITLE
Make `highway=living_street` unsearchable in Bulgaria

### ISSUE THREAD

**Issue by Dimitar5555** [openstreetmap/id-tagging-schema#1367]
In Bulgaria very few streets can be tagged with `highway=living_street` (probably less than five). The problem is that the preset's label is literally translated to "Residential Road" which confuses mappers, because `highway=residential` has the same translation.

Is it possible to make `highway=living_street` unsearchable only in Bulgaria?

**Comment by Dimitar5555** [comment]
`highway=residential` is translated as "Улица" ("Street")

`highway=living_street` is translated as "Улица в жилищна зона" ("Street in Residential zone", where residential zone is supposed to have a special meaning but it also means regular residential zone).

`landuse=residential` is translated as "Жилищна зона" ("Residential zone") which could be use for regular residential zone, as well as "residential zone with living streets".

Also see two of the many changesets where mappers tag `highway=residential` streets as living streets even when they arent. :

https://www.openstreetmap.org/changeset/158401601
https://www.openstreetmap.org/changeset/158400569

This is why I think that the best solution is to hide the preset.

**Comment by Oivo35** [comment]
I think hide `highway=living_street` is not a good idea. [жилищна зона](https://bg.wikipedia.org/wiki/%D0%A4%D0%B0%D0%B9%D0%BB:Bulgaria_road_sign_%D0%9415.svg) (residential area zone) is regular road sign in Bulgaria.
![256px-Bulgaria_road_sign_Д15 svg](https://github.com/user-attachments/assets/9f4d16a8-2438-44f4-ac18-9b4946712bba)
`highway=residential` can be  `zhilishtna ulitsa`,
`highway=living_street` can be `zhilishtna zona` - only for ways,
`landuse=residential` can be also `zhilishtna zona` - only for nodes and areas.

**Comment by matkoniecz** [comment]
> "Street in Residential zone"

maybe turn it into

> "Street in Residential zone defined by a special sign"

or similar?

**Comment by Dimitar5555** [comment]
> I think hide highway=living_street is not a good idea. [жилищна зона](https://bg.wikipedia.org/wiki/%D0%A4%D0%B0%D0%B9%D0%BB:Bulgaria_road_sign_%D0%9415.svg) (residential area zone) is regular road sign in Bulgaria.

There is a single living street in the whole country. Every other instance of `highway=living_street` is incorrect.

> > "Street in Residential zone"
> 
> maybe turn it into
> 
> > "Street in Residential zone defined by a special sign"
> 
> or similar?

I've just changed it to `Улица в жилищна зона (обозначена със пътен знак Д15)` (`Street in a residential area (signposted with road sign D15)`) although I'm not sure if it will help.

**Comment by matkoniecz** [comment]
> There is a single living street in the whole country.

which one is actually existing?

**Comment by Dimitar5555** [comment]
Tsar Ivan Shishman in Sofia, starting from https://www.openstreetmap.org/way/683840371 and going in NE direction ([Gmaps](https://www.google.com/maps/@42.6893916,23.327444,3a,81y,59.58h,94.12t/data=!3m7!1e1!3m5!1sF9sJbR720RABGgqltLZsWA!2e0!6shttps:%2F%2Fstreetviewpixels-pa.googleapis.com%2Fv1%2Fthumbnail%3Fcb_client%3Dmaps_sv.tactile%26w%3D900%26h%3D600%26pitch%3D-4.124137870040286%26panoid%3DF9sJbR720RABGgqltLZsWA%26yaw%3D59.581898783083794!7i16384!8i8192?coh=205410&entry=ttu&g_ep=EgoyMDI0MTAyOS4wIKXMDSoASAFQAw%3D%3D) for reference, there is no other street level imagery that shows it, the signs were placed this year).

**Comment by tordans** [comment]
Given there seems to be an official sign https://github.com/openstreetmap/id-tagging-schema/issues/1367#issuecomment-2440035573 I think the way to go is to find a better translation.

I suggest to keep this issue open until someone edits the translations so you can use it to discuss better wordings.

**Comment by Dimitar5555** [comment]
> [@Dimitar5555](https://github.com/Dimitar5555) Have you tried updating translations?

Yes, I did. As expected, it had no effect since most people use English as their locale. Even people who used Bulgarian locale ignored the giant text.

<img width="552" height="253" alt="Image" src="https://github.com/user-attachments/assets/e12616ae-2d4f-4e0e-896b-5316f3b0bf37" />

https://osm.org/way/1459512069
https://osm.org/way/1463888661
